cw/tvla.py

```diff
@@ -883,12 +883,16 @@ def run_tvla(ctx: typer.Context):
         else:
             #
             c = np.ones(num_samples)
+            xaxs = range(sample_start, sample_start + num_samples)
             fig, axs = plt.subplots(3, sharex=True)
-            axs[0].plot(single_trace, "k")
+            axs[0].set_title(
+                "TVLA of " + cfg["project_file"] + '\n' + "No. of traces: " + str(num_traces))
```

Comment:
I think just this line here actually belongs into this very commit.

Reply:
Hmm let's put it also in the other PR, where we change the plotting